src/tests/electron/tests/first-time-dialog.test.ts

```diff
@@ -21,14 +21,8 @@ describe('first time dialog', () => {
         }
     });
 
-    it.each([true, false])(
-        'should pass accessibility validation with highContrastMode=%s',
-        async highContrastMode => {
-            await app.setHighContrastMode(highContrastMode);
-            await app.waitForHighContrastMode(highContrastMode);
-
-            const violations = await scanForAccessibilityIssues(dialog);
```

Comment:
dialog is still defined on line 10 but never used, which is causing lint to fail :(